### PR TITLE
Mark 2.0.30 as released

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

**Comment by dschuff** [comment]
I forgot to bump emscripten-version.txt last time, so it gets 2 bumps this time.

File: ChangeLog.md [hunk 1]
```diff
@@ -18,8 +18,11 @@ to browse the changes between the tags.
...
 See docs/process.md for more on how version tagging works.
 
+2.0.31
+------
 2.0.30
 ------
+- Bug fixes
 2.0.29
 -----
 - Bug fixes
```
Comment (kripken) [inline ChangeLog.md:28]:
```suggestion
- Bug fixes

```

File: ChangeLog.md [hunk 2]
```diff
@@ -18,8 +18,11 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
+2.0.31
+------
 2.0.30
 ------
+- Bug fixes
```
Comment (kripken) [inline ChangeLog.md:25]:
```suggestion
- Bug fixes

```

File: ChangeLog.md [hunk 3]
```diff
@@ -18,8 +18,11 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
+2.0.31
+------
 2.0.30
```
Comment (kripken) [inline ChangeLog.md:23]:
```suggestion
2.0.30

```

**Review by kripken — APPROVED** [review]
Added comments for more whitespace, wdyt?

**Comment by dschuff** [comment]
I don't think the whitespace affects the Markdown rendering, is this just for readability of the md file itself?

**Comment by kripken** [comment]
Yes, just readability of the md. We used to have more spacing there, but looks like the last few releases had less - I'm not sure why.

**Comment by dschuff** [comment]
Probably just because I was too lazy to add it :D

**Pull request merged**